nautobot/circuits/templates/circuits/circuit_retrieve.html

```diff
@@ -14,7 +14,7 @@
                 <tr>
                     <td>Status</td>
                     <td>
-                        <span class="label" style="color: {{ object.status.color|fgcolor }}; background-color: #{{ object.status.color }}">{{ object.get_status_display }}</span>
+                        {{ object | render_obj_display:"status" }}
```

Comment:
Instead of `render_obj_display` perhaps something like `render_related_name_and_color` would be clearer?